tests/third_party/test_numpy.py

```
@@ -12,15 +12,15 @@
 
 from unittest import TestCase
 
+import numpy as np
```

Pylint (3.3.7)

[error] Unable to import 'numpy'

(E0401)